src/components/teamMember.scss

```diff
@@ -105,9 +110,24 @@
   max-height: 1280px;
 }
 
+.contact-details {
+  margin-top: var(--spacing-s);
+}
+
+.contact-details p {
+  font-family: var(--visueltRegular);
+  font-size: var(--paragraphFontSize);
+  line-height: 17px;
+  margin-bottom: 12px;
+}
+
+.contact-details a {
+  color: var(--ocean);
+}
+
 @media only screen and (max-width: 889px) {
   .team-member {
```

Comment:
I think we lost some styles when you refactored this to support both the about page and the contact page. On mobile, the contact page team members are overflowing and not wrapping onto the next line. Can we make sure to match the mobile layout of the About page here?

![image](https://user-images.githubusercontent.com/22242017/102110435-c455ae80-3dfa-11eb-92c2-6bfb14102c66.png)